pulp-glue/tests/test_api_quirks.py

```diff
@@ -1,27 +1,18 @@
-import typing as t
 from copy import deepcopy
 
 import pytest
+
 from pulp_glue.common.context import _REGISTERED_API_QUIRKS, PulpContext
 
+pytestmark = pytest.mark.glue
```

Comment:
Is this another pytest quirk? How does it work?

Reply:
It marks all tests in this scope. (Putting it in conftest did not work sadly.)